### DISCUSSION TITLE
playback speed

### DISCUSSION THREAD

**Discussion by Nikhil-sharma-c** [SubtitleEdit/subtitleedit#9019] [Q&A]
while using the text to speech api in a video . the voice generated is adjust according to the video length changing it playback speed which makes voice inconsistence and fast . is their anyway to control voice playback speed before merging in or to increase length of video and then merge it ?

**Comment by niksedk** [comment]
The best way is probably to make sure your CPS is not too high, and if it's too high, either give it more time or make the text shorter.

**Comment by Nikhil-sharma-c** [comment]
Thankyou sir
